### PR TITLE
Content for Q2

### PR DESCRIPTION
**Pull request by bnaolu**
- Added examples of violations and resources.
- Also added resources to the "Right to Access Notice" for requesting health info.

### CODE REVIEW

**Comment by bnaolu** [comment]
Some (hopefully helpful) notes I took for this content:

- HIPAA Privacy Rule: https://www.hhs.gov/hipaa/for-professionals/privacy/index.html
The main goal of this rule is to guarantee that an individual’s health information is well protected but within a system that still lets the information flow between the parties that need PHI in order to provide the best quality of healthcare to the patient. The Privacy Rule also says that patients should have access to the same information about themselves that their doctors do and they should get some level of authority over where that information goes and who has access to it. 
- HIPAA Security Rule: https://www.hhs.gov/hipaa/for-professionals/security/laws-regulations/index.html
Ensuring patient’s electronic health information is adequately protected and is only available to authorized users and not improperly accessed or used. Covered entities (CEs) and their Business Associates (BAs) must have three safeguards in place: administrative, physical, and technical.
- Patient Safety Act and Rule: 
A voluntary system through which providers could share sensitive information relating to patient safety events without fear of liability, which should lead to improvements in patient safety and in the quality of patient care. 
- Breach Notification Rules: https://www.hhs.gov/hipaa/for-professionals/breach-notification/index.html
If a breach occurs, CEs and BAs must notify the affected patients.

- HHS All Case examples
 https://www.hhs.gov/hipaa/for-professionals/compliance-enforcement/examples/all-cases/index.html

- HHS Enforcement Highlights
https://www.hhs.gov/hipaa/for-professionals/compliance-enforcement/data/enforcement-highlights/index.html 

- Common Examples of HIPAA Violations
https://examples.yourdictionary.com/examples-of-hipaa-violations.html
https://www.hipaajournal.com/common-hipaa-violations/
  - **Impermissible uses and disclosures of protected health information**;
 - A healthcare worker who was not directly involved with your treatment looks into your files. 
  - Healthcare workers discuss about your health information in a common area where non-authorized personnel could easily overhear. 
  - A nurse or medical professional releases your health information to a party that is not formally authorized to receive the data.
  - **Lack of safeguards of protected health information**;
   - Your health information was sent through a public fax line or through unencrypted emails. 
   - Your health information was not properly disposed of in order to prevent a violation.
    - A healthcare worker forgets to lock or log off their computers that contain patient health information.
     - A healthcare worker leaves patient health information visible on a computer screen while others can see it. 
  - **Lack of patient access to their protected health information**;
   - Your provider denies you access to your health records or has exceeded the time period of providing access. 
  - **Lack of administrative safeguards of electronic protected health information**; 
   - Failure to implement appropriate ePHI access controls
    - Failure to use encryption, or an equivalent measure, on portable devices with PHI 
   - Exceeding the 60-day deadline for issuing breach notifications 
  - **Use or disclosure of more than the minimum necessary protected health information**.
   - A nurse needs access to private health information for the patients in his/her unit but not for any patients that are not in that unit. 
   - A health insurance company will need information about the number of visits the customer had; but, isn’t allowed to view the entire patient history 
   - The security guard in a healthcare institution needs to know the name and room number of patients to guide visitors. This is allowed; but, any other information, such as diagnosis or treatment, is not to be disclosed.

- Additional Examples (simplified?)
  - telling friends or relatives about patients in the hospital, doctors office or treatment facility when you work
  - discussing patients or PHI in public areas of the hospital, including the lobby of a hospital, an elevator or the cafeteria
  - discussing patients or PHI over the phone in a public area
  - not logging off your computer or a computer system that contains PHI
  - allowing members of the media to interview a patient in a substance abuse facility
  - posting images to social media that could potentially include patient likenesses without specific written permission to do so
  - sharing images to social media in which PHI is in any way visible

File: question-2.md [hunk 1]
```diff
@@ -8,16 +8,42 @@ type: radio
...
+                    </li>
+                    <li>Your health information was not properly disposed of in order to prevent a violation.
+                    </li>
+                    <li>A healthcare worker leaves your patient health information visible on a computer screen while others can see it.
+                    </li>
+                    <li>A healthcare organization or those directly involved with them, exceeded the 60-day deadline for issuing breach notifications.
+                    </li>
+                </ul>
         </ul>
+        <h2>Resources</h2>
```
Comment (pglevy) [inline question-2.md]: This should be an h3 since it's a "child" of an h2.

File: question-2.md [hunk 2]
```diff
@@ -8,16 +8,42 @@ type: radio
 <div class="usa-accordion margin-bottom-2">
     <h2 class="usa-accordion__heading">
         <button class="usa-accordion__button" aria-expanded="false" aria-controls="a1">
-            See health information rights
+            See examples of common health information violations
         </button>
     </h2>
     <div id="a1" class="usa-accordion__content">
         <ul>
```
Comment (pglevy) [inline question-2.md:16]: Can we add a single line to preface these examples, something to the effect of "here are some examples, but there are other situations." Not those words, something better, but hopefully avoids the feedback of "I didn't see my specific issue in the list."

**Review by pglevy — CHANGES_REQUESTED** [review]

**Comment by bnaolu** [comment]
Changes made. Review when ready 😊

**Review by pglevy — APPROVED** [review]

**Pull request merged**